src/components/AddChore/AddChore.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import "./AddChore.css";
+import { useState } from "react";
+
+function AddChore({ onAddChore }) {
+  const [chore, setChore] = useState({
+    choreName: "",
+    dueDate: "",
+    assignee: "",
+    frequency: "",
+    reminder: "",
+  });
+
+  const submitButton = (e) => {
+    e.preventDefault();
+    onAddChore({ ...chore });
+
+    setChore({
+      choreName: "",
+      dueDate: "",
+      assignee: "",
+      frequency: "",
+      reminder: "",
+    });
+  };
+
+  return (
+    <div>
+      <h2>Add a Chore</h2>
+      <form className="add-chore-card add-chore-scroll" onSubmit={submitButton}>
+        <label htmlFor="name" className="add-chore-info">
+          Name:
+          <input
+            className="add-chore-input"
+            data-testid="input-choreName"
+            id="name"
+            type="text"
+            placeholder="Chore Name"
+            value={chore.choreName}
+            onChange={(e) => {
+              setChore({ ...chore, choreName: e.target.value });
+            }}
+          />
+        </label>
+        <label htmlFor="dueDate" className="add-chore-info">
+          Due:
+          <input
+            className="add-chore-input"
+            data-testid="input-dueDate"
+            id="dueDate"
+            type="date"
+            value={chore.dueDate}
+            onChange={(e) => {
+              setChore({ ...chore, dueDate: e.target.value });
+            }}
+          />
+        </label>
+        <label htmlFor="assignee" className="add-chore-info">
+          Assignee:
+          <input
+            className="add-chore-input"
+            data-testid="input-assignee"
+            id="assignee"
+            type="text"
+            placeholder="Assignee"
+            value={chore.assignee}
+            onChange={(e) => {
+              setChore({ ...chore, assignee: e.target.value });
+            }}
+          />
+        </label>
+        <label htmlFor="frequency" className="add-chore-info">
+          Frequency:
+          <input
+            className="add-chore-input"
+            data-testid="input-frequency"
+            id="frequency"
+            type="number"
```

Comment:
I'm assuming frequency + reminders are placeholders for now? Because saying "frequency = 2" doesn't make sense vs 2 days, weeks, etc. I think reminders make a little more sense right now, but we might want to allow for multiple reminders in the future and/or figure out how many reminders to send if it's overdue and things like that.